### DISCUSSION TITLE
Having issues with application

### DISCUSSION THREAD

**Discussion by gittoit** [community/community#137957] [GitHub Education]
### Hi everyone,

Hi, Im currently a student and my ID doesnt have a date on it. Am I able to upload a screenshot to verify my status as a student?

**Comment by andrefqms** [comment]
Hi @gittoit 

I assume you need to prove your academic status to apply for the GitHub Student Developer pack. If that's the case, continue reading on.

With regards to your application for GitHub Student Developer pack, you can consider submitting documents such as the following:

Picture of your school ID with current enrollment date
Academic transcript
School affiliation or enrollment verification letter
Class schedule

If all fails, please submit a ticket to Github Education Support over [here](https://support.github.com/contact/education)
